pandas/tests/arrays/test_integer.py

```diff
@@ -826,6 +826,38 @@ def test_astype_nansafe():
         arr.astype("uint32")
 
 
+@pytest.mark.parametrize(
+    "values, dtype",
+    [
+        ([9999999999999999, 123123123123123123, 10000000000000543, np.nan], "Int64"),
+        (
+            np.array(
+                [9999999999999999, 123123123123123123, 10000000000000543],
+                dtype=np.int64,
+            ),
+            "Int64",
+        ),
+    ],
+)
+def test_construction_large_numbers(values, dtype):
```

Comment:
dtype looks like a constant, don't use in the parameterization, hard code inside the test instead